file1.js

```diff
@@ -0,0 +1,8 @@
+/* global QamineUtils */
+
+"use strict";
+
+(function($, window) {
+  $('.jsSelectedMainEmail').text(postVariable.email);
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/prodcodacytestuser/CheckPrCommentsPublicProjectProd/pullRequest?prid=7701107)
```suggestion
  $(".jsSelectedMainEmail").text(postVariable.email);
```